packages/runtime/container-runtime/src/summary/summarizerTypes.ts

```diff
@@ -320,10 +304,9 @@ export interface ISummarizerEvents extends IEvent {
 	(event: "summarizingError", listener: (error: ISummarizingWarning) => void);
 }
 
-export interface ISummarizer
-	extends IEventProvider<ISummarizerEvents>,
-		IFluidLoadable,
-		Partial<IProvideSummarizer> {
+export interface ISummarizer extends IEventProvider<ISummarizerEvents> {
```

Comment:
Should we document this type while we're here? Feel free to disregard, since this was not documented before. But it might be helpful for anyone reading the changelog to have a better understanding of what this interface is.